app/controllers/idv/inherited_proofing_cancellations_controller.rb

```diff
@@ -82,5 +78,10 @@ def flow_step_allowlist
     def effective_user_id
       current_user&.id
     end
+
+    def analytics_properties
+      route_properties = ParseControllerFromReferer.new(request.referer).call
+      route_properties.merge({ analytics_id: @analytics_id })
+    end
```

Comment:
Might want to move this above the overridden methods (marked by comments) as it looks like an overridden method; or, just remove all those "...overrides..." comments altogether.

Reply:
You want me to remove the comments that you added?

Reply:
Sure if it makes it easier.